### DISCUSSION TITLE
PR tab: Option to view PRs in my projects

### DISCUSSION THREAD

**Discussion by tripleee** [community/community#90203] [Mobile]
When I tap the Pull Requests item on the home page in Github Mobile, I expect to see (or at least be able to select a view which allows me to see) open PRs in ropositories I own.

Currently, I can only pick PRs which are assigned to me, created by me, requesting my review, or etc; there is no way to enable a view where none of these selectors is active, even if I search to focus on a specific repo.

Acutely, I received an email about a new PR, and wanted to leave a comment  on it; but it turns out that navigating to it from the Pull Requests top-level view is simply not possible in the Github Mobile IOS app, which I consider counter-intuitive to the point of being annoying.

The (in retrospect) obvious workaround is to visit that repository, and take it from there; but clearly, there are other use cases where that's unattractive - I would regard it as obvious that I should be able to view "my" PRs, across repositories, even when they are not explictly assigned to me or etc.

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.145.0 (120653769)
Device: iPhone 12 mini (iOS 17.2.1)
</details>

**Comment by eliperkins** [comment]
Hey @tripleee, thanks for this feedback! I'd love to learn more from you. Are you able to see the "Repositories" filter in the filter bar when scrolling the available filters? It should look something like this:

https://github.com/community/community/assets/1051453/101be881-8583-48eb-b922-15ca5150224c

Selecting a repository in the filter bar should let you focus on only a specific repo. After selecting a repo, you can also create a shortcut for that filter by selecting the filter button at the beginning of the filter bar! With shortcuts, you can save a set of configured filters, right to the Home tab for quick access

![IMG_DA0E34B075A9-1](https://github.com/community/community/assets/1051453/bcc7b486-e13b-409a-b36b-41bb9acbd53b)


I hope this helps. Let me know if I've misunderstood what your goals are. I'd love to learn more. 🙇

**Reply by tripleee** [reply]
It's there alright, but that doesn't remove the "Created by me" filter which is there by default, and can be changed to e.g. "Involved", but not removed, which means I cannot reveal new unassigned issues which don't mention me.
